site/en/docs/extensions/mv3/geolocation/index.md

```diff
@@ -51,9 +51,9 @@ After adding the `offscreen` permission, add an html file to your extension that
 <script src="offscreen.js"></script>
 ```
 
-Save this file in the root of our project, as `offscreen.html`.
+Save this file in the root of your project as `offscreen.html`.
 
-As mentioned you need a script called `offscreen.js`. You'll also need to bundle this with your extension. It will be where you actually get geolocation information, and send that information back to the service worker. You can send and receive messages between it and your service worker.
+As mentioned, you need a script called `offscreen.js`. You'll also need to bundle this with your extension. It will be where you actually get geolocation information and send that information back to the service worker. You can send and receive messages between it and your service worker.
```

Comment:
```suggestion
As mentioned, you need a script called `offscreen.js`. You'll also need to bundle this with your extension. It will be the service worker's source of geolocation information. You can pass messages between it and your service worker.
```